app/models/support_interface/providers_filter.rb

```diff
@@ -105,6 +112,17 @@ def filter_records(providers)
         providers = providers.left_outer_joins(:provider_permissions).where(provider_users_providers: { id: nil })
       end
 
+      if applied_filters[:accredited_provider].present?
+        accrediting_provider_ids = Provider
+                                    .where("CONCAT(providers.name, ' ', providers.code) ILIKE ?", "%#{applied_filters[:accredited_provider]}%")
+                                    .map(&:id)
+
+        providers = providers
+                    .distinct
+                    .joins(courses: :accredited_provider)
+                    .where(courses: { accredited_provider_id: accrediting_provider_ids })
```

Comment:
any thoughts on whether this should be scoped to a particular cycle? Just thinking eventually this is going to become less useful when many cycles have passed. Maybe there also needs to be a courses from x year filter so they can be used in conjunction?

Reply:
thinking about this i think it makes sense to scope it to the current cycle as all the onboarding work will take place at the launch of the next cycle anyway

Reply:
I would guess it makes sense to scope to the current cycle.